inc/saiswitch.h

```diff
@@ -2695,6 +2695,23 @@ typedef enum _sai_switch_attr_t
      */
     SAI_SWITCH_ATTR_IPSEC_SA_STATUS_CHANGE_NOTIFY,
 
+    /**
+     * @brief Number of ECMP members supported by My MAC entries installed on the switch
+     *
+     * @type sai_uint32_t
+     * @flags READ_ONLY
+     */
+    SAI_SWITCH_ATTR_MAX_ECMP_MEMBER_COUNT,
+
+    /**
+     * @brief Number of  ECMP Members configured on the switch
+     *
+     * @type sai_uint32_t
+     * @flags CREATE_AND_SET
+     * @default 64
+     */
+    SAI_SWITCH_ATTR_ECMP_MEMBER_COUNT,
+
```

Comment:
The meaning of this is not clear.  

1) Is this applicable for all types of sai_next_hop_group_type_t?

2) Can this attribute be made a per Next Hop Group attribute instead of a switch attribute?

3) Can you also see if we can relax SAI_NEXT_HOP_GROUP_ATTR_CONFIGURED_SIZE to be applicable for other next_hop_group types?

Reply:
1. This is a switch level attribute that helps carve out memory for max ecmp members and ecmp groups. Nothing to do with next group configuration.
2. No, there is different attribute to control nexthop count in nexthop group : SAI_NEXT_HOP_GROUP_ATTR_NEXT_HOP_COUNT
3. I believe we can but that is beyond the scope of this PR

Reply:
how is the ECMP_MEMBER_COUNT related to https://github.com/opencomputeproject/SAI/blob/master/inc/sainexthopgroup.h#L160 ?